pkg/analytics/util.go

```diff
@@ -0,0 +1,48 @@
+package analytics
+
+import (
+	otellog "go.opentelemetry.io/otel/log"
+
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+func makeJobAttributes(j models.Job) []otellog.KeyValue {
+	return []otellog.KeyValue{
+		otellog.String("id", j.ID),
+		otellog.String("name", j.Name),
+		otellog.String("namespace", j.Namespace),
+		otellog.String("type", j.Type),
+		otellog.Int("count", j.Count),
+		// TODO(forrest): consider collecting constraints, meta, and labels
+		otellog.String("state", j.State.StateType.String()),
+		otellog.String("state_message", j.State.Message),
+		otellog.Int64("version", int64(j.Version)),
+		otellog.Int64("revision", int64(j.Revision)),
+		otellog.Int64("create_time", j.CreateTime),
+		otellog.Int64("modified_time", j.ModifyTime),
+	}
+}
+
+func makeTaskAttributes(t *models.Task) []otellog.KeyValue {
+	inputTypes := make([]otellog.Value, len(t.InputSources))
+	for i, s := range t.InputSources {
+		inputTypes[i] = otellog.StringValue(s.Source.Type)
+	}
+
+	return []otellog.KeyValue{
+		otellog.String("task_name", t.Name),
+		otellog.String("task_engine", t.Engine.Type),
+		otellog.String("task_publisher", t.Engine.Type),
+		otellog.Slice("task_inputs", inputTypes...),
+		otellog.Int("task_env_count", len(t.Env)),
+		otellog.Int("task_meta_count", len(t.Meta)),
```

Comment:
I think it could be helpful to get a count of all meta data - we know how many we expect to set, so any value over that could be interesting to collect. Additionally, we we see `0` here we know there is probably a bug in our code. 

I'll leave the decision up to you.

Reply:
I am okay with whatever you decide